config/routes.rb

```diff
@@ -29,6 +29,7 @@
   get '/for_organizations/recruiting', to: 'home#recruiting', as: :recruiting
   get '/wieso-myzivi', to: 'home#for_civil_servants', as: :for_civil_servants
   get '/about_us', to: 'home#about_us'
+  get '/zivi-faq', to: 'home#civil_servant_faq'
```

Comment:
```suggestion
  get '/zivi-faq', to: 'home#civil_servant_faq', as: :civil_servant_faq
```

and then adapt helper method to `civil_servant_faq_path`